images.yaml

```diff
@@ -228,6 +228,18 @@
 - name: k8s.gcr.io/hyperkube
   patterns:
   - pattern: '>= v1.15'
+- name: k8s.gcr.io/kube-apiserver
+  patterns:
+  - pattern: '>= v1.16'
```

Comment:
Should we make it `>= 1.17` or do you want to test a transition with these?

Reply:
I was thinking it would be cool to make this change in 1.16 so we're prepared for 1.17. I guess we really only need 1.16.9+ in that case though.

Reply:
I see - let's go with `1.16.9+` then